molequeue/app/CMakeLists.txt

```diff
@@ -1,11 +1,9 @@
-set(qt_components Core Widgets Network)
+find_package(Qt5 REQUIRED COMPONENTS Core Widgets Network)
 
 if(MoleQueue_USE_EZHPC_UIT)
-  list(APPEND qt_components XmlPatterns)
+  find_package(Qt5XmlPatterns REQUIRED)
```

Comment:
I only want to make calls to find_package(Qt5 ...) with components, this enables us to only set Qt5_DIR rather than a bunch of Qt5ModuleName_DIR variables.